_posts/2022-06-02-configure-tls-with-lets-encrypt-ca.adoc

```diff
@@ -0,0 +1,9 @@
+---
+layout: post
+title:  'Vlog: Configure TLS in WildFly using Let's Encrypt certificate authority'
```

Comment:
The singal quote in Let's needs swapping to use the same quote which is used on line 6 - as this is terminating the line early.  Alternatively the outer quotes on this line can be swapped with double quotes "